lib/config/config.go

```diff
@@ -37,6 +37,7 @@ var (
 	// config.
 	DefaultListenAddresses = []string{
 		"tcp://0.0.0.0:22000",
+		"kcp://0.0.0.0:22020",
```

Comment:
Not sure, but for example "port 22000 in use" can mean either, hence less confusion?